sunset/code/modules/reagents/chemistry/machinery/reagentgrinder.dm

```diff
@@ -0,0 +1,15 @@
+
+#define MILK_TO_BUTTER_COEFF 15
```

Comment:
wtf is that define?

Reply:
I have no idea, that probably accidental got caught in when I was making the sub type.